hack/ci-e2e.sh

```diff
@@ -126,6 +136,8 @@ docker run --name image-server-e2e -d \
 set +e
 
 # Run the e2e tests
+export E2E_CONF_FILE="${REPO_ROOT}/test/e2e/config/ironic.yaml"
+export USE_EXISTING_CLUSTER="true"
```

Comment:
These 2 lines are like this by default, isn't it?

Reply:
Yes. The assumption is that since you run `ci-e2e.sh`, this is what you want. Of course it's just to be extra careful, nothing would happen in CI if we take it away.